src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -93,6 +99,25 @@ public function getTransport(): ?string
         return $this->transport;
     }
 
+    /**
+     * @return $this
+     */
+    public function from(?string $from): static
+    {
+        if ('' === $from) {
+            throw new InvalidArgumentException(sprintf('"%s" needs a valid "from", it cannot be empty.', static::class));
+        }
+
+        $this->from = $from;
+
+        return $this;
+    }
+
+    public function getFrom(): ?string
```

Comment:
Not know how to deal with new getter.
I think I have to create `SmsMessageInterface` but introduce BC Break